pkg/operator/starter.go

```diff
@@ -72,7 +72,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		kubeInformersForNamespaces,
 		assets.ReadFile,
 		[]string{
-			"volumesnapshotclass.yaml",
+			//"volumesnapshotclass.yaml", //TODO: snapshots are disabled until restore is available: https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/136
```

Comment:
I'd also suggest completely removing this file.

Reply:
Done.